.github/workflows/dev-smoke-test.yml

```
@@ -0,0 +1,97 @@
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install Lite UI deps
+        working-directory: webui_lite
+        run: npm ci
+
+      - name: Generate build timestamp
+        id: timestamp
+        run: echo "datetime=$(date -u +%Y%m%d_%H%M%S)" >> $GITHUB_OUTPUT
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:1:46: Double quote to prevent globbing and word splitting

(shellcheck)